consensus/core/src/storage/rocksdb_store.rs

```diff
@@ -15,32 +16,35 @@ use typed_store::{
     Map as _,
 };
 
-use super::Store;
-use crate::commit::{CommitAPI as _, TrustedCommit};
+use super::{CommitInfo, Store, WriteBatch};
+use crate::commit::{CommitAPI as _, CommitDigest, TrustedCommit};
 use crate::{
-    block::{BlockDigest, BlockRef, Round, SignedBlock, VerifiedBlock},
-    commit::{Commit, CommitIndex},
+    block::{BlockAPI as _, BlockDigest, BlockRef, Round, SignedBlock, VerifiedBlock},
+    commit::CommitIndex,
     error::{ConsensusError, ConsensusResult},
 };
 
 /// Persistent storage with RocksDB.
 pub(crate) struct RocksDBStore {
     /// Stores SignedBlock by refs.
-    blocks: DBMap<(Round, AuthorityIndex, BlockDigest), bytes::Bytes>,
+    blocks: DBMap<(Round, AuthorityIndex, BlockDigest), Bytes>,
     /// A secondary index that orders refs first by authors.
     digests_by_authorities: DBMap<(AuthorityIndex, Round, BlockDigest), ()>,
     /// Maps commit index to content.
-    // TODO: Use Bytes for value. Add CommitDigest to key.
-    commits: DBMap<CommitIndex, Commit>,
-    /// Stores the last committed rounds per authority.
-    last_committed_rounds: DBMap<(), Vec<Round>>,
+    commits: DBMap<(Round, CommitIndex, CommitDigest), Bytes>,
```

Comment:
Oh yeah it wasn't a matter of store cost per se, but rather semantics and expectations when looking on the key. Ok that makes sense. For the peer-certified commits why we expect the digest to change though? Whatever authenticators used for the certification should they be part of the digest so this would change from one authority to the other?